### PR TITLE
Add keymaps for muralis and kuru

### PR DESCRIPTION
**Pull request by muralisc**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dz60/keymaps/muralisc/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dz60/keymaps/muralisc/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/dztech/dz60rgb_ansi/keymaps/kuru/keymap.c [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_ansi/keymaps/kuru/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/dztech/dz60rgb_ansi/keymaps/kuru/keymap.c [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    rgb_matrix_set_color(34, 0x00, 0x00, 0xFF);
+    // PgDwn, End
+    rgb_matrix_set_color(36, 0x00, 0x99, 0x00);
+    rgb_matrix_set_color(46, 0x00, 0x99, 0x00);
+}
+
+void rgb_matrix_indicators_user(void) {
+	  uint8_t this_led = host_keyboard_leds();
+      if (!g_suspend_state) {
+        switch (biton32(layer_state)) {
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_ansi/keymaps/kuru/keymap.c:74]:
```suggestion
        switch (get_highest_layer(layer_state)) {
```

File: keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/keymap.c [hunk 4]
```diff
@@ -0,0 +1,152 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/keymap.c [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+    int easy_arrow_keys[4] = {41, 55, 54, 53};
+    for(int i  = 0; i < 4; i++) {
+      rgb_matrix_set_color(easy_arrow_keys[i], 0x09, 0x00, 0x09);
+    }
+}
+
+void rgb_matrix_indicators_user(void) {
+	  uint8_t this_led = host_keyboard_leds();
+      if (!g_suspend_state) {
+        switch (biton32(layer_state)) {
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/keymap.c:135]:
```suggestion
        switch (get_highest_layer(layer_state)) {
```

File: keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/Readme.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+### Flashing
+
+Set keyboard to Reset mode and 
+
+New Method
+```
+CFLAGS="-Wno-error=deprecated" qmk compile -kb dztech/dz60rgb_ansi/v1 -km muralisc 
+CFLAGS="-Wno-error=deprecated" qmk flash -kb dztech/dz60rgb_ansi/v1 -km muralisc
+```
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/Readme.md:36]: Not a fan of this.  Shouldn't need to use the cflags stuff.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/dz60/keymaps/muralisc/keymap.c (hunk 1 above)
Comment (muralisc) [reply keyboards/dz60/keymaps/muralisc/keymap.c:1]: Sure, thanks for taking out time to review

File: keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/keymap.c (hunk 4 above)
Comment (muralisc) [reply keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/keymap.c:17]: added

File: keyboards/dz60/keymaps/muralisc/keymap.c (hunk 1 above)
Comment (muralisc) [reply keyboards/dz60/keymaps/muralisc/keymap.c:17]: added

File: keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/Readme.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+### ansi vim
```
Comment (drashna) [inline keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/Readme.md:1]: Please rename this file to be all lower case.

Comment (muralisc) [reply keyboards/dztech/dz60rgb_ansi/keymaps/muralisc/Readme.md]: done

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**